javascript/javascript2/week3/fastestPresser.js

```diff
@@ -0,0 +1,33 @@
+
+let pressS=0;
+    let pressL=0;
+    let miliSec=0;
+    let btn=document.getElementById('btn');
+    btn.addEventListener("click",()=>{
+        miliSec=document.getElementById('seconds').value*1000;
+        let body=document.querySelector('html');
+        body.addEventListener("keydown",letterCounter);
```

Comment:
I would have totally done the same choice, but it would still be the wrong one. Keydown means that if the user will hold S or L .. it will spam the score.. The right one is key up.